mobile/src/main/AndroidManifest.xml

```diff
@@ -87,6 +87,23 @@
             android:exported="false" />
         <receiver android:name=".shared.notifications.CancelNotificationBroadcastReceiver"
             android:exported="false" />
+
+        <provider
+            android:name="androidx.startup.InitializationProvider"
+            android:authorities="${applicationId}.androidx-startup"
+            android:exported="false"
+            tools:node="merge">
+            <meta-data
+                android:name="com.google.samples.apps.iosched.util.initializers.AndroidThreeTenInitializer"
+                android:value="androidx.startup" />
+            <meta-data
+                android:name="com.google.samples.apps.iosched.util.initializers.StrictModeInitializer"
+                android:value="androidx.startup" />
+            <!-- Initalize TimberInitializer after the Application's onCreate is called -->
+            <meta-data
+                android:name="com.google.samples.apps.iosched.util.initializers.TimberInitializer"
+                tools:node="remove" />
```

Comment:
To lazily initialize `Timber` after the app is created. [Lazy initialization](https://developer.android.com/topic/libraries/app-startup#manual) is suggested to reduce startup cost.